model-view-viewmodel/pom.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>java-design-patterns</artifactId>
+		<groupId>com.iluwatar</groupId>
+		<version>1.24.0-SNAPSHOT</version>
+	</parent>
+	<groupId>com.iluwatar</groupId>
+	<artifactId>model-view-viewmodel</artifactId>
+	<version>1.24.0-SNAPSHOT</version>
+	<properties>
+		<zk.version>9.0.0</zk.version>
+		<guava.version>19.0</guava.version>
```

Comment:
The guava version already exists in the root pom over [here](url)

I will suggest you to remove it.

https://github.com/iluwatar/java-design-patterns/blob/b5ed8b227857f0b446157d43fa98814630922444/pom.xml#L53

Reply:
@ohbus, As mentioned in above comment same is the case with guava.
If I remove it, the individual module doesn't compile.
I can't remove it from parent either as its being used by other modules as well.

Reply:
I believe the code should be able to meet the dependency requirements from the parent pom just like all other modules in this repository does.

I will follow it up after we merge the code.